### ISSUE TITLE
 Update account description and transaction type description to pull from MDM

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2345]
### Definition of Ready
- [x] Does this meet the INVEST criteria?
- [x] Does this have a clearly defined user?
- [x] Does this have an estimate?
- [x] Does this have acceptance criteria?
- [x] Does this have sufficient context for the specific functionality being developed (design, business rules, etc.)
- [x] Does this have a dev link been provided in the design?
- [x] Is this traceable to a feature, epic, or PI?

## User Story
As a CAMPD user I want the account description to pull from the MDM table so that the data is consistent 

## Acceptance Criteria
Given that a user has made a GET request to the allowance holdings, account info, or allowance transactions endpoint 
When the request is executed 
Then the system returns the appropriate data elements
AND the data elements adhere to the [Master data dictionary](https://usepa.sharepoint.com/:x:/r/sites/CAMDCVPTeam/_layouts/15/Doc.aspx?sourcedoc=%7BE47D193A-AD4E-491A-A2AC-6EFD4DB50FBC%7D&file=Final_CAMPD_Master%20Data%20Labels%20List_EPA%20CAMD_v1.0.xlsx&action=default&mobileredirect=true) 

### Development Notes
Update the allowance holdings, account info, and  transactions endpoint account descriptions field to be pulled from the MDM table instead of the account fact/transaction fact tables

Update transaction type for allowance transactions. For testing ensure that the endpoints are still returning the correct data elements and that OTC and NBP data is returning 

Dependency: Need to wait on the database to be refreshed. The db will be refreshed during the weekend of 1/21. Cannot conduct manual testing until the db has been refreshed  

### Does this Story require 508 Testing?
- [ ] Implementing a new authoring tool that will produce code / content for the application
- [ ] Implementing a 3rd party widget that will be part of the application
- [ ] Adding new / updating custom functionality 
- [ ] Adding new / updating a page template
- [ ] Adding new / replacing existing functionality (unless the replacement has already passed 508 testing)
- [ ] Full Site Review prior to a scheduled EPA 508 review
- [ ] Planning a video that will appear on the application
- [ ] Producing a video that will appear on the application
- [ ] Producing Training materials
- [ ] Producing documents

## Testing
Test Case: [ TC531 ](https://epacamd.testquality.com/project/11428/plan/20171/test/321348)
Tester: @JanellC 

### Definition of Done
- [x] Have all Acceptance Criteria been agreed to and validated by the scrum team?
- [x] Has the code been unit tested and peer reviewed?
- [x] Have the functional tests been executed?
- [x] Have all defects been reviewed and dispositioned, resolved, or deferred?
- [x] Is the user story ready to be deployed to the test/staging environments?

**Comment by JanellC** [comment]
TC531 Passed @tlageman-epa 

**Allowance Transactions** 

https://api.epa.gov/easey/dev/account-mgmt/allowance-transactions?api_key=T8r7OW4f12XtWKLY7CPgKmKu1WoP3TVG0x4eqmlB&page=3000&perPage=50&programCodeInfo=NBP|OTC&transactionBeginDate=1993-03-23&transactionEndDate=2018-03-30

https://api.epa.gov/easey/dev/account-mgmt/allowance-transactions?api_key=T8r7OW4f12XtWKLY7CPgKmKu1WoP3TVG0x4eqmlB&page=1&perPage=50&programCodeInfo=NBP|OTC&transactionBeginDate=1993-03-23&transactionEndDate=2018-03-30

**Allowance holdings**

https://api.epa.gov/easey/dev/account-mgmt/allowance-holdings?api_key=T8r7OW4f12XtWKLY7CPgKmKu1WoP3TVG0x4eqmlB&page=1&perPage=1

**Account info**

https://api.epa.gov/easey/dev/account-mgmt/accounts/attributes?api_key=T8r7OW4f12XtWKLY7CPgKmKu1WoP3TVG0x4eqmlB&page=1&perPage=100&programCodeInfo=NBP|OTC
